CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.3] - 2020-07-22
```

Comment:
link ref is missing at end of file for `[0.2.3]`

Reply:
fixed in 523bf38

Reply:
resolved in 523bf38